src/Npgsql/Internal/NpgsqlDatabaseInfo.cs

```diff
@@ -194,19 +219,25 @@ internal void ProcessTypes()
         /// </summary>
         protected static Version ParseServerVersion(string value)
         {
-            var versionString = value.Trim();
+            var versionString = value.AsSpan().TrimStart();
```

Comment:
No 😆
I tend to spanify everything these days because it doesn't hurt - until .NET Standard gets in my way.
I can move it back to the simple version if you want, although I have to admit that I'm personally offended by an allocation Version of `Trim()` nowadays.

Reply:
I'll leave it up to you. If it were up to me, since there's zero perf impact here, I'd go for whatever code is simplest.

Reply:
Ok, I'll mostly revert it to the original version and only keep the `TrimStart()` optimization and the bugfix (did you actually notice the bug in the previous code?).

Reply:
Ah no, I didn't notice that - good catch.